app/modules/topics/components/Editor/index.js

```diff
@@ -38,6 +53,13 @@ const mapDispatchToProps = (
     onSave: (): void => {
       dispatch(actions.patchWithContent(topicId));
     },
+    setDirty: (dirty: boolean): void => {
+      dispatch(actions.setDirtyInState(topicId, dirty));
+    },
+    discard: (): void => {
```

Comment:
Event handlers that are passed in as props should be prefixed with `on`; i.e. `onSetDirty`, `onDiscard`, etc.